.env.template

```diff
@@ -0,0 +1,21 @@
+# .env file
+################################################################################
```

Comment:
can we add some directions here that tells the user to copy this into their own .env file?